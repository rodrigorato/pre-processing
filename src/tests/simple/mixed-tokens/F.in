public class Foo {
	alias display = System.out.println;
	alias var = String[];
	public static void main(var args) {
		var s = new var() { "One", "Two", "Three"};

		for (int i = 0; i < 3; i++)
			display(#"{s[i]}");
	}
}