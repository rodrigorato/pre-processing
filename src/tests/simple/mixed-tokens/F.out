public class Foo {
	
	
	public static void main(String[] args) {
		String[] s = new String[]() { "One", "Two", "Three"};

		for (int i = 0; i < 3; i++)
			System.out.println("{s[i]}");
	}
}